.github/workflows/integration.yaml

```diff
@@ -2,6 +2,23 @@ name: Test Suite
 on: [pull_request]
 
 jobs:
+  static-analysis:
+    name: Static analysis
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.8]
+    steps:
```

Comment:
I thought it was needed because we have a step
```yaml
    - name: Install dependencies
      run: python3 -m pip install tox
```
but it works without. Removed.